features/attr.yml

```
@@ -0,0 +1,19 @@
+name: attr
```

For consistency with other CSS notation features like this:

```suggestion
name: attr()
```